crates/uv/src/commands/pip/list.rs

```diff
@@ -150,26 +150,48 @@ pub(crate) async fn pip_list(
         results
     };
 
+    let verbose = true; // TODO: this should be a flag
```

Comment:
How do we thread this through to the list command, and do we still want the verbosity flag to apply to `uv` in general?